openml/study/study.py

```diff
@@ -89,6 +89,36 @@ def __init__(
         self.runs = runs
         pass
 
+    def __str__(self):
+        object_dict = self.__dict__
+        output_str = ''
+        id = '\n%-16s: %s\n' % ("ID", object_dict['id'])
+        name = '%-16s: %s\n' % ("Name", object_dict['name'])
+        status = '%-16s: %s\n' % ("Status", object_dict['status'])
+        main_entity_type = '%-16s: %s\n' % ("Main Entity Type", object_dict['main_entity_type'])
+        url = 'https://www.openml.org/s/' + str(object_dict['id'])
+        study_url = '%-16s: %s\n' % ("Study URL", url)
+        data = ''
+        if object_dict['data'] is not None:
+            data = '%-16s: %s\n' % ("# of Data", len(object_dict['data']))
+        tasks = ''
+        if object_dict['tasks'] is not None:
+            tasks = '%-16s: %s\n' % ("# of Tasks", len(object_dict['tasks']))
+        flows = ''
+        if object_dict['flows'] is not None:
+            flows = '%-16s: %s\n' % ("# of Flows", len(object_dict['flows']))
+        runs = ''
+        if object_dict['runs'] is not None:
+            runs = '%-16s: %s\n' % ("# of Runs", len(object_dict['runs']))
+
+        url = 'https://www.openml.org/u/' + str(object_dict['creator'])
+        creator = '\n%-16s: %s\n' % ("Creator", url)
+        upload_time = '%-16s: %s\n' % ("Upload Time",
+                                       object_dict['creation_date'].replace('T', ' '))
+        output_str = id + name + status + main_entity_type + study_url + data + \
```

Comment:
What do you think about adding a header to the output string, for example 
```
OpenML Dataset
==============
ID: 5 (version 1)
...
```
obviously, this needs to be adapted for the different entities. Also, you'd need to create separate functions for the study and the suite class.

Reply:
I have added __str__ functions for the other two classes which inherit this class, OpenMLStudy and OpenMLBenchmarkSuite. The __str__ appends the relevant header to the base class' __str__ string.